dockstore-webservice/src/main/templates/io/dockstore/webservice/resources/Description.java

```diff
@@ -35,15 +36,18 @@
 import static io.dockstore.webservice.Constants.JWT_SECURITY_DEFINITION_NAME;
 
 /**
- * This is a dummy class used to describe the swagger API as a whole
+ * This is a dummy class used to describe the swagger API as a whole.
+ * WARNING: This is a generated file, do not edit directly.
+ * Edit the template in src/main/templates
```

Comment:
You could list the full path, `src/main/templates/io/dockstore/webservice/resources/Description.java`, but it's not hard to figure out, so not crucial